src/edge_op.cpp

```diff
@@ -95,6 +95,33 @@ struct FlagEdge {
   }
 };
 
+struct AntiparallelEdge {
+  VecView<const Halfedge> halfedge;
+  VecView<const vec3> triNormal;
+  const int firstNewVert;
+  const double
+      angleThreshold;  // Threshold for considering normals antiparallel
+
+  bool operator()(int edge) const {
+    const Halfedge& half = halfedge[edge];
+    if (half.pairedHalfedge < 0 || !half.IsForward()) return false;
+    // Only process edges where at least one vertex is new
+    if (half.startVert < firstNewVert && half.endVert < firstNewVert)
+      return false;
+
+    // Check if the adjacent triangle normals are antiparallel (180 degrees
+    // apart)
+    const int tri1 = edge / 3;
+    const int tri2 = half.pairedHalfedge / 3;
+    const double dotProduct = la::dot(triNormal[tri1], triNormal[tri2]);
+
+    // If normals are pointing in opposite directions (dot product close to -1)
+    // This indicates a sliver where triangles are nearly coplanar but facing
+    // opposite directions
+    return dotProduct < angleThreshold;  // e.g., -0.99 for nearly antiparallel
```

Comment:
I wonder if our `CreateFaces()` function should handle this case? I'm a little surprised it doesn't currently...